modules/apps/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss

```diff
@@ -30,6 +30,7 @@ html#{$cadmin-selector} {
 			z-index: $product-menu-zindex;
 
 			&.open,
+			&.open-admin-panel,
```

Comment:
We don't need this here. It's already covered in `analytics-reports-web/.../main.scss`.

Reply:
I remember having trouble making it work without `open-admin-panel` I think that some of the panel is missing the right CSS. So we can either find it and add what's missing or try to fix it using a common CSS class (like `lfr-product-menu-panel` ) and remove the duplicated code. I've found at least 3 different yet similar cases:
- https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/css/main.scss
- https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss
- https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/main.scss

Reply:
It worked for me without this when I deployed it.